diff --git a/examples/network-manager-app/network-manager-common/network-manager-app.zap b/examples/network-manager-app/network-manager-common/network-manager-app.zap
index 98816f9319..0dd177ee4e 100644
--- a/examples/network-manager-app/network-manager-common/network-manager-app.zap
+++ b/examples/network-manager-app/network-manager-common/network-manager-app.zap
@@ -224,24 +224,7 @@
           "define": "ACCESS_CONTROL_CLUSTER",
           "side": "server",
           "enabled": 1,
-          "commands": [
-            {
-              "name": "ReviewFabricRestrictions",
-              "code": 0,
-              "mfgCode": null,
-              "source": "client",
-              "isIncoming": 1,
-              "isEnabled": 1
-            },
-            {
-              "name": "ReviewFabricRestrictionsResponse",
-              "code": 1,
-              "mfgCode": null,
-              "source": "server",
-              "isIncoming": 0,
-              "isEnabled": 1
-            }
-          ],
+          "commands": [],
           "attributes": [
             {
               "name": "ACL",
diff --git a/src/platform/logging/impl/Syslog.cpp b/src/platform/logging/impl/Syslog.cpp
index 638cffeb94..70650f5544 100644
--- a/src/platform/logging/impl/Syslog.cpp
+++ b/src/platform/logging/impl/Syslog.cpp
@@ -15,6 +15,8 @@
  *    limitations under the License.
  */
 
+#include <cstdio>
+
 #include <platform/logging/LogV.h>
 
 #include <lib/core/CHIPConfig.h>

